crates/bevy_render/src/color/mod.rs

```diff
@@ -47,11 +47,15 @@ pub enum Color {
     },
 }
 
+// List of HTML colors sourced from https://www.w3schools.com/tags/ref_colornames.asp.
```

Comment:
This is the wrong link - please re-derive based on https://www.w3.org/TR/css-color-4/#named-colors. In theory, I think we also need to re-derive the list from that source. I'm not sure how that works when the list would be the same here, but when moving them between modules, please re-derive them. Thanks

(The w3schools page is all rights reserved, so we can't use it.)